### PR TITLE
Added error page heading

### PR DESCRIPTION
**Pull request by yleaxeman**

### CODE REVIEW

**Review by proand — COMMENTED** [review]
I think we should make a fhi-error-page block instead of fhi-error-page-heading block. Yes, there will be an "unnecessary" wrapper element, but the block get more future proof.

File: src/fhi/blocks/_fhi-error-page.block.scss [hunk 1]
```diff
@@ -0,0 +1,20 @@
+.fhi-error-page {
+    &__heading {
+        @extend .h2;
+        margin-bottom: $fhi-core-space-3;
+
+        &__status-code {
```
Comment (proand) [inline src/fhi/blocks/_fhi-error-page.block.scss:6]: No need to nest this one into heading

**Review by proand — CHANGES_REQUESTED** [review]

File: src/fhi/blocks/_fhi-error-page.block.scss (hunk 1 above)
Comment (yleaxeman) [reply src/fhi/blocks/_fhi-error-page.block.scss:6]: Sure? I'd say it is dependant on the .fhi-error-page__heading

Comment (yleaxeman) [reply src/fhi/blocks/_fhi-error-page.block.scss:6]: Ah, I see the BEM info documentation. Got it.

**Review by proand — APPROVED** [review]

**Pull request merged**